tools/tsp-client/src/commands.ts

```diff
@@ -412,3 +413,82 @@ export async function sortSwaggerCommand(argv: any): Promise<void> {
   await writeFile(swaggerFile, JSON.stringify(sorted, null, 2));
   Logger.info(`${swaggerFile} has been sorted.`);
 }
+
+export async function combineSwaggersCommand(argv: any): Promise<void> {
+    Logger.info("Combining multiple swagger files into a single file...");
+    const outputDir = argv["output-dir"];
+    const swaggerFiles = argv["swagger-files"];
+    const swaggerFileCount = swaggerFiles?.length ?? 0;
+    if (swaggerFileCount <= 1) {
+        throw new Error(`2 or more swagger files must be specified, ${swaggerFileCount} provided.`);
+    }
+
+    await ensureDirectory(outputDir);
+    
+    const documents = [];
+    for (const file of swaggerFiles) {
+        if (!(await doesFileExist(file))) {
+            throw new Error(`Swagger file not found: ${file}`);
+        }
+        try {
+            const content = await readFile(file);
+            const document = JSON.parse(content.toString());
+            documents.push(document);
+        } catch (err) {
+            throw new Error(`Failed to parse swagger file: ${file}. Error: ${err}`);
+        }
+    }
+
+    // Merge contents 1 level deep
+    const combined = documents.reduce((a: Record<string | number, any>, b: Record<string | number, any>) => {
+        const result = { ...a };
+        for (const [key, value] of Object.entries(b)) {
+            // if the key doesn't exist in result, just set it to b's value
+            if (!(key in result || typeof result[key] === "undefined" || result[key] === null)) {
+                result[key] = value;
+                continue;
+            }
+    
+            // when b value is null and any a value exists, ignore b value
```

Comment:
Isnt this a regression if we add the value of a back into the combined swaggers?